packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -318,6 +396,7 @@ class _DraggableScrollableSheetState extends State<DraggableScrollableSheet> {
           curve: Curves.linear,
         );
       }
+      _extent.hasChanged = false;
```

Comment:
Is this right? Dependencies can change after a user has interacted with the sheet.

Reply:
Ahh ok, I see now. The way the diff was rendered by default was confusing me.